pkg/agent/flannel/setup.go

```diff
@@ -68,6 +68,12 @@ const (
 	"PSK": "%psk%"
 }`
 
+	tailscaledBackend = `{
+	"Type": "extension",
+	"PostStartupCommand": "tailscale up --accept-routes --advertise-routes=$SUBNET",
```

Comment:
user is expected to provide tailscale binaries on their own, I take it?

Reply:
Yes! That will be explained in the docs. It is explained in the ADR: "At this stage, the proposal is not to incorporate the vpn binaries or daemons inside the k3s binary, we require the user to have the vpn binary installed."

And also important: "The collection of vpn endpoint information and its start are implemented by calling the vpn binary. Tailscale has a "not-feature-complete" go client but netmaker does not, so calls to the binary is the common denominator"